source/games/game.py

```diff
@@ -59,3 +59,6 @@ def player_joined(self, player_name):
     def broadcast(self, message):
         for sock in self.sockets:
             sock.send(message.serialize())
+
+    def does_player_exiset(self,sock):
```

Comment:
The function name has a typo. I think you meant "does_player_exist" but I think "does_sock_exist" would be more fitting